db/migrate/20181011143410_create_addresses.rb

```diff
@@ -0,0 +1,13 @@
+class CreateAddresses < ActiveRecord::Migration[5.2]
+  def change
+    create_table :addresses do |t|
+      t.string :address_line_one, null: false
```

Comment:
Why is `address_line_one` set to `null: false`, but the other parameters that are validated in the model for presence (`address_line_two`, `city`, `postcode`) not set the same way. I'd suggest removing the `, null: false` from `address_line_one` and letting the model control this.

Reply:
postcode did have this as well but I have remove it from both now